public/app/percona/shared/components/PerconaBootstrapper/PerconaBootstrapper.tsx

```diff
@@ -39,9 +41,16 @@ export const PerconaBootstrapper = () => {
       }
     };
 
-    getSettings();
-    getUserStatus();
-    dispatch(fetchServerInfoAction());
+    const bootstrap = async () => {
+      await getSettings();
```

Comment:
Do we handle errors inside those methods or do we want to do it here?

Reply:
Most of them are being handled in the reducers, using the `withSerializedError` util from Grafana.
`getSettings` and `getUserStatus` are being handled here but they should go to the reducers. I have a PR with that but I think it's better to wait for 2.27.0 before doing that change